src/util/dlist/Mybuild

```diff
@@ -1,13 +1,12 @@
 package embox.util
 
 
-@DefaultImpl(DListDebug)
-abstract module DList {}
+@DefaultImpl(dlist_debug)
+abstract module dlist {}
 
-static module DListDebug extends DList {
+static module dlist_debug extends dlist {
 	source "dlist_debug.h"
 	source "dlist_debug.c"
-}
-static module DListNDebug extends DList {
-	source "dlist_ndebug.h"
+
+	option number dlist_enable_debug=1
```

Comment:
Options are defined localy for each module, so it's ok to use shorter name like `debug` or something like that.

Reply:
OK